cardano-node/cardano-node.cabal

```diff
@@ -202,6 +202,7 @@ executable chairman
                      , containers
                      , cardano-api
                      , cardano-config
+                     , cardano-ledger
```

Comment:
@dcoutts shouldn't this be the ledger specs repo?

Reply:
It's the `cardano-ledger` package that lives in the ledger specs repo in `byron/ledger/impl/`

Reply:
Indeed